meson.build

```diff
@@ -1,14 +1,44 @@
-#
-# build script written by : Michael Brockus.
-# github repo author: Mike Karlesky, Mark VanderVoord, Greg Williams.
-#
-# license: MIT
-#
 project('unity', 'c',
-    license: 'MIT',
-    meson_version: '>=0.53.0',
-    default_options: ['werror=true', 'c_std=c11']
+  version: run_command(
+    find_program('python'), 'auto' / 'extract_version.py', 'src/unity.h',
+    check: true).stdout(),
+  license: 'MIT',
+  meson_version: '>=0.53.0',
+  default_options: ['werror=true', 'c_std=c11']
 )
 
+build_fixture = get_option('extension_fixture').enabled()
+build_memory = get_option('extension_memory').enabled()
```

Comment:
You're just using these like booleans, it seems misleading to allow people to specify either `=auto` or `=disabled` and have auto really just mean disabled.

What about defining the options as type boolean, and just checking the option value directly?